src/Agent.Listener/JobDispatcher.cs

```diff
@@ -466,7 +466,19 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, WorkerDisp
                     // send notification to machine provisioner.
                     var systemConnection = message.Resources.Endpoints.SingleOrDefault(x => string.Equals(x.Name, WellKnownServiceEndpointNames.SystemVssConnection, StringComparison.OrdinalIgnoreCase));
                     var accessToken = systemConnection?.Authorization?.Parameters["AccessToken"];
-                    await notification.JobStarted(message.JobId, accessToken, systemConnection.Url);
+                    VariableValue buildId = null;
```

Comment:
Can we use a generic name? One that represents buildId and releaseId

Reply:
It's on us to write the queries backing it - would be good to have a single *Type* and *Id*. If a new planType is introduced for pipelines, the code should still make sense.